PWGHF/TableProducer/candidateSelectorDplusToPiKPi.cxx

```diff
@@ -117,6 +118,7 @@ struct HfCandidateSelectorDplusToPiKPi {
         hfMlResponse.setModelPathsLocal(onnxFileNames);
       }
       hfMlResponse.init();
+      hfMlResponse.cacheInputFeaturesIndices(namesInputFeatures);
       outputMl.assign(((std::vector<int>)cutDirMl).size(), -1.f); // dummy value for ML output
```

Comment:
Sorry, what do you mean by keeping it empty?

Reply:
Hi @AlexBigO, with no entries instead of dummy values. The column that is filled is a variable-length array so can have different sizes for each entry and the vector could also be empty.